components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -43,9 +43,21 @@ type Props = {
     stats: any;
     theme: any;
     usersLimit: number;
+    actions: {
+        getTotalUsersStats: () => any;
+    };
 }
 
 export default class ChannelIntroMessage extends React.PureComponent<Props> {
+    showAddMemberButton = false;
+
+    componentDidMount() {
+        if (this.props.actions.getTotalUsersStats) {
```

Comment:
Should this be checking if the data is loaded in the state (`props.stats`) vs checking the existence of the action?

Reply:
ok!